gradle.properties

```diff
@@ -14,7 +14,7 @@
 # limitations under the License.
 #
 
-ideaEdition = IC
+ideaEdition = IU
```

Comment:
could you please update this? i'm open to changing the default value, but not in this pr.